### DISCUSSION TITLE
[Bug] 'schannel:  CertGetCertificateChain trust error CERT_TRUST_IS_UNTRUSTED_ROOT'

### DISCUSSION THREAD

**Discussion by Agentofchaoss** [ScoopInstaller/Scoop#5896] [Help and Questions]
# Bug Report:

Nothing was changed or edited prior to this error. Was working perfectly fine until the other day when I booted up my system and went to run the update when I received this error in response, please help me. I am completely perplexed as to why this happened. Looking forwasrd to hearing back as soon as possible. Thank you for your time.

## Behavior:
**Current:**
1. *Cannot execute the following commands:*
 - `sudo scoop update --all`
 - `scoop update --all`
 - `scoop update`
2. *Cannot find https://github.com/ScoopInstaller/ Buckets*

**Expected:**
 1. update as normal❌
 2. Autoupdate as normal❌

## Additional context/output:
 - `"WARN  Uncommitted changes detected. Update aborted."`
 - `"CertGetCertificateChain trust error CERT_TRUST_IS_UNTRUSTED_ROOT"`
 - `"fatal: unable to access"`

![2023-11-09](https://github.com/ScoopInstaller/Scoop/assets/109058188/68d099b5-da35-4860-9773-df88fe5f0782)

### System Info:
 - **Windows version:** `10pro`
 - **OS architecture:** `64bit`
 - **PowerShell version:** `PowerShell 7.3.9`
 - **Additional software:** `Git && Tabby terminal`

#### Scoop Configuration:

```json
{
  "last_update": "2023-11-09T17:13:00.4854392-08:00",
  "root_path": "~:\\Users\\admin\\scoop",
  "global_path": "~:\\ProgramData\\scoop",
  "aria2-enabled": false,
  "scoop_branch": "master",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by Agentofchaoss** [comment]
Need help with this issue please and thank you.

**Comment by HUMORCE** [comment]
> "WARN  Uncommitted changes detected. Update aborted."

Restore all changes you have made to Scoop Core(`scoop prefix scoop`) and bucket repos(`scoop bucket list` to check the location).

Other errors are related to other settings of PC, try searching for them.
